q2_moshpit/kraken2/select.py

```diff
@@ -162,9 +165,16 @@ def _kraken_to_ncbi_tree(df):
 
     # last entry is always a tip
     _, parent_node = stack[-1]
+    # It is possible for the last row to be an infra-clade tip,
+    # so walk backwards up the stack until a standard node (with length)
+    # is found
+    while stack and parent_node.length == 0:
+        _, parent_node = stack.pop()
+
     if parent_node.children:
         parent_node.children[0].is_actual_tip = True
 
+    print(tree)
```

Comment:
Well. I'll need to fix that. Expect another PR from me @misialq, but hopefully with a working CI.